test/jasmine/specs/global-object-css-spec.js

```diff
@@ -12,6 +12,24 @@ describe("Test getting JSON from <body/> CSS", function () {
         );
     });
 
+    it("should get expected strings from global element", function () {
```

Comment:
Probably not necessary, I was just trying to isolate the string-only portion of sass-to-js. The failing tests appeared to be unrelated (they weren't able to find _any_ styles on the expected objects). Did you manage to get these tests to pass in your other commit?